updating link in pages/dkp/kommander/1.3/install/index.md (#3416)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.3/install/index.md

Prerequisites for Kommander are the same as those for Konvoy. See the [Konvoy Prerequisites](/dkp/konvoy/1.7/quick-start/#prequisites) for information.

# Download and Install

> Kommander is provided by Konvoy by default as part of [Kubernetes Base Addons](/dkp/konvoy/1.7/addons/). See the [Download Konvoy](/dkp/konvoy/1.7/download/) topic for more information on how to download Konvoy. Once Konvoy is downloaded, see [Installing Konvoy](/dkp/konvoy/1.7/quick-start/#installing-konvoy) for instructions on how to install it. Once Konvoy is installed with the default settings, Kommander is automatically provided on the cluster.

Reviewer: ```suggestion
By default, Konvoy provides Kommander as part of [Kubernetes Base Addons](/dkp/konvoy/1.7/addons/). See the [Download Konvoy](/dkp/konvoy/1.7/download/) topic for more information on how to download Konvoy. Once Konvoy is downloaded, see [Installing Konvoy](/dkp/konvoy/1.7/quick-start/#installing-konvoy) for installation instructions. After Konvoy is installed with the default settings, Kommander is automatically provided on the cluster.
```
